### ISSUE TITLE
Cleanup TODO

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/arkimet#165]
[TODO](https://github.com/ARPA-SIMC/arkimet/blob/master/TODO) file is a mix of abandoned or already implemented issues, changelog-style comments and so on.

Valid issues should be migrated to github, historical notes should be deleted or moved in some different file.

**Comment by spanezz** [comment]
I did some TODO cleanup, opened #174, and these are the only TODO bits left. I'm removing the TODO file and moving them here for discussion:

```
 - spostare --merged lato server (e farlo solo tra dataset che stanno sullo
   stesso server)
    - Mergia il possibile lato server, poi rimergia lato client se piglia da
      server diversi
       - Se si rimergia lato client, rifare il sort
```

this is mostly included in #114 and #153.

```
 - TimeIntervalSorter: permettere di configurarlo in modo che se ci sono 2 dati
   con lo stesso reftime e altri metadati, permetta di scegliere se darli in
   output tutti o se dare in output solo quello del primo dataset che ce l'ha
   (solo nel caso di multidataset, va rivisto a cosa serve, perché se serve
   come precondizione al postprocessing, non si facomunque postprocessing nel
   multidataset)
   Caso d'uso: quero i modelli multidataset, l'operativo e il backup. Se non
   c'è il dato dell'operativo, lo prendo dal backup, ma se c'è il dato in
   entrambi, voglio solo il primo.
```

This is a new use case for querying. Is it needed?

```
 - Manpage 'arkimet' con:
    - un'introduzione generale
    - descrizione e puntatori alle manpage dei vari comandi
    - puntatore alla manpage con la sintassi del matcher
    - puntatore alla manpage con la sintassi dei file di configurazione e degli
      alias
 - Aggiungere agli errori un link al wiki
 - Grafici di come sono processati i dati a seconda di vari comandi di
   arki-query e arki-scan
```

Is this needed, or is it already covered by the existing arkimet documentation produced in ARPA in the last years?

**Comment by brancomat** [comment]
Those TODO bits are either implemented satisfactorily or old feature requests that apparently don't matter anymore.
Since the last comment is almost 1y old, I'm closing the issue, we could open new feature requests when really needed.
